JoomGalleryfriends/JG4-dev
# Apply access service to backend views

This PR applies the access service provided with PR #113 to the JoomGallery MVC backend classes. This means that in the backend the advanced `own` and `inown` permissions are working now.

Additionally I improved the code style and coding notes of the backend PHP files. And all easy to replace deprecations are replaced like `Factory::getDbo;` with `Factory::getContainer()->get(DatabaseInterface::class);`.

### How to test this PR
Set error reporting mode to maximum in the Joomla global configuration.

All views and actions should work as before and should not throw any errors.
Additionally check that users without super user rights can access and perform tasks according to the ACL settings set in the global component setting and in the corresponding form of the content type (image form if you want to create an image ...).
